modules/dr-restoring-cluster-state.adoc

```diff
@@ -314,3 +314,29 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 ----
 
 Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+[id="dr-scenario-cluster-state-issues_{context}"]
+= Issues and workarounds for restoring a persistent storage state
+
+If your {product-title} cluster uses persistent storage of any form, a state of the world is typically stored outside etcd. It might be an Elasticsearch cluster running in a pod or a database running in a `StatefulSet`object. When restoring from an etcd backup, the status of the workloads in {product-title} is also restored. However, if the etcd snapshot is old, the status might be invalid or outdated.
```

Comment:
Please add a space between `StatefulSet` and "object".

nit: Instead of "When restoring..." how do you feel about "When you restore..."?